python/tests/test_utils.py

```diff
@@ -0,0 +1,20 @@
+import json
+import os
+import unittest
+
+from fds.protobuf.stach.extensions.StachExtensionFactory import StachExtensionFactory
+from fds.protobuf.stach.extensions.StachVersion import StachVersion
+from fds.protobuf.stach.extensions.v2.StachUtilities import StachUtilities
+
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+class V2StachUtilsTests(unittest.TestCase):
+    def test_stach_utils_v2(self):
+        column_values = ["Port.+Weight","Bench.+Weight","Bench.+Weight","Bench.+Weight"]
+        file = os.path.join(ROOT_DIR, "resources", "V2StachUtils.json")
+        with open(file) as f:
+            data = json.load(f)
+            decompressed_data = StachUtilities.decompress_column(data["data"]["columns"]["col1"])
```

Comment:
so do we need to decompress each column manually, will it be better to have a utility that decompress all the columns? just a thought, can discuss on pros and cons

Reply:
There is a second function `decompress_page` added with this change that would address the use case of decompressing all relevant columns. Is there a different use case you are referring to?